lightningd/peer_control.c

```diff
@@ -407,6 +408,15 @@ void channel_errmsg(struct channel *channel,
 					     err_for_them,
 					     tal_count(err_for_them), 0);
 
+	/* Other implementations chose to ignore errors early on.  Not
+	 * surprisingly, they now spew out spurious errors frequently,
+	 * and we would close the channel on them. */
+	if (soft_error) {
+		channel_fail_transient(channel, "%s: (ignoring) %s",
```

Comment:
`channel_fail_transient` now has `slow_reconnect` flag, which seems, not included here.

I seem, the format string is auto-convert to bool, then `channel->owner->name` is treated as format string.  And, since there is no test for this functionality, it is not found this bug in our CI...

Reply:
Yes, I hit the same issue elsewhere. I'm going to rename the function to fix this properly, so it breaks like it should